solr/core/src/test/org/apache/solr/cli/TestSolrCLIRunExample.java

```diff
@@ -102,7 +102,7 @@ public int execute(org.apache.commons.exec.CommandLine cmd) throws IOException {
       commandsExecuted.add(cmd);
 
       String exe = cmd.getExecutable();
-      if (exe.endsWith("solr")) {
+      if (exe.endsWith("solr") || exe.endsWith("solr.cmd")) {
```

Comment:
somethign I am confused by....   How would the exe ever NOT be either solr or solr.cmd???   Is there a way this code can be run that doesn't include one of those?  I am just wondering if we need this if statement?

Reply:
i looked at the class...  and it's super specific to testing Solr CLI, so I can't imagine a world where anything BUT a solr or solr.cmd is used???   In fact, I think an assert would be more apprporiate, to fail the test if you used this class with anything but solr or solr.cmd?

Reply:
Love it!   Thanks for the explanation and the simplification.  I love elminating if loops.